src/test/java/com/wikia/webdriver/testcases/desktop/createawikitests/CreateWikiTestsLang.java

```diff
@@ -25,8 +25,14 @@ public void createNewWikiLangTC001(String lang) {
     WikiBasePageObject base = new WikiBasePageObject();
     CreateNewWikiPageObjectStep1 cnw1 = base.openSpecialCreateNewWikiPage(wikiCorporateURL);
     cnw1.selectLanguage(lang);
-    String expectedDomainPrefix = "en".equals(lang) ? "" : String.format("%s.", lang);
-    assertEquals(cnw1.getDomainPrefix().getText(), expectedDomainPrefix);
+    String expectedDomainSufix;
+    if (lang.equals("en")) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Move the "en" string literal on the left side of this string comparison. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1132)